CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.22.1 (2024-09-11)
+This is a re-release of 1.22.0, please refer to the release note for 1.22.0 for detailed release content.
```

Comment:
I suggest this:

This is a re-release of 1.22.0. Please refer to the 1.22.0 release notes for detailed release content.

Reply:
updated